spec/spec_helper.rb

```diff
@@ -15,6 +18,18 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
+  config.before(:each) do
+    stub_request(:get, "https://api.github.com/user/orgs").
```

Comment:
It's OK for now — but I think using https://github.com/vcr/vcr will be easier to scale when our number of tests grows.